--- a/net/minecraft/server/Main.java
+++ b/net/minecraft/server/Main.java
@@ -152,14 +152,14 @@
 
          session.backupLevelDataFile(impl, saveProperties);
          SaveProperties saveProperties2 = saveProperties;
-         final MinecraftDedicatedServer minecraftDedicatedServer = MinecraftServer.startServer((serverThread) -> {
+         final MinecraftDedicatedServer minecraftDedicatedServer2 = MinecraftServer.startServer((serverThread) -> {
             MinecraftDedicatedServer minecraftDedicatedServer = new MinecraftDedicatedServer(serverThread, impl, session, resourcePackManager, serverResourceManager, saveProperties2, serverPropertiesLoader, Schemas.getFixer(), minecraftSessionService, gameProfileRepository, userCache, WorldGenerationProgressLogger::new);
             minecraftDedicatedServer.setServerName((String)optionSet.valueOf(optionSpec9));
             minecraftDedicatedServer.setServerPort(optionSet.valueOf(optionSpec12));
             minecraftDedicatedServer.setDemo(optionSet.has(optionSpec3));
             minecraftDedicatedServer.setServerId((String)optionSet.valueOf(optionSpec13));
-            boolean bl = !optionSet.has(optionSpec) && !optionSet.valuesOf(optionSpec14).contains("nogui");
-            if (bl && !GraphicsEnvironment.isHeadless()) {
+            boolean bl2 = !optionSet.has(optionSpec) && !optionSet.valuesOf(optionSpec14).contains("nogui");
+            if (bl2 && !GraphicsEnvironment.isHeadless()) {
                minecraftDedicatedServer.createGui();
             }
 
@@ -167,7 +167,7 @@
          });
          Thread thread = new Thread("Server Shutdown Thread") {
             public void run() {
-               minecraftDedicatedServer.stop(true);
+               minecraftDedicatedServer2.stop(true);
             }
          };
          thread.setUncaughtExceptionHandler(new UncaughtExceptionLogger(LOGGER));
@@ -208,3 +208,5 @@
 
    }
 }
+
+// Fukkit Marker
