--- a/net/minecraft/server/MinecraftServer.java
+++ b/net/minecraft/server/MinecraftServer.java
@@ -1246,7 +1246,7 @@
     * @param datapacks a collection of datapacks to reload with
     */
    public CompletableFuture<Void> reloadResources(Collection<String> datapacks) {
-      CompletableFuture<Void> completableFuture = CompletableFuture.supplyAsync(() -> (ImmutableList)datapacks.stream().map(this.dataPackManager::getProfile).filter(Objects::nonNull).map(ResourcePackProfile::createResourcePack).collect(ImmutableList.toImmutableList()), this).thenCompose((immutableList) -> ServerResourceManager.reload(immutableList, this.isDedicated() ? CommandManager.RegistrationEnvironment.DEDICATED : CommandManager.RegistrationEnvironment.INTEGRATED, this.getFunctionPermissionLevel(), this.workerExecutor, this)).thenAcceptAsync((serverResourceManager) -> {
+      CompletableFuture<Void> completableFuture = CompletableFuture.supplyAsync(() -> datapacks.stream().map(this.dataPackManager::getProfile).filter(Objects::nonNull).map(ResourcePackProfile::createResourcePack).collect(ImmutableList.toImmutableList()), this).thenCompose((immutableList) -> ServerResourceManager.reload(immutableList, this.isDedicated() ? CommandManager.RegistrationEnvironment.DEDICATED : CommandManager.RegistrationEnvironment.INTEGRATED, this.getFunctionPermissionLevel(), this.workerExecutor, this)).thenAcceptAsync((serverResourceManager) -> {
          this.serverResourceManager.close();
          this.serverResourceManager = serverResourceManager;
          this.dataPackManager.setEnabledProfiles(datapacks);
@@ -1648,3 +1648,5 @@
       return null;
    }
 }
+
+// Fukkit Marker
