--- a/net/minecraft/server/ServerConfigList.java
+++ b/net/minecraft/server/ServerConfigList.java
@@ -146,7 +146,7 @@
                JsonObject jsonObject = JsonHelper.asObject(jsonElement, "entry");
                ServerConfigEntry<K> serverConfigEntry = this.fromJson(jsonObject);
                if (serverConfigEntry.getKey() != null) {
-                  this.map.put(this.toString(serverConfigEntry.getKey()), serverConfigEntry);
+                  this.map.put(this.toString(serverConfigEntry.getKey()), (V) serverConfigEntry);
                }
             }
          } catch (Throwable var15) {
@@ -170,3 +170,5 @@
       }
    }
 }
+
+// Fukkit Marker
