--- a/net/minecraft/server/ServerConfigHandler.java
+++ b/net/minecraft/server/ServerConfigHandler.java
@@ -307,10 +307,10 @@
                }
 
                private void convertPlayerFile(File playerDataFolder, String fileName, String uuid) {
-                  File file = new File(file, fileName + ".dat");
-                  File file2 = new File(playerDataFolder, uuid + ".dat");
+                  File filex = new File(file, fileName + ".dat");
+                  File file2x = new File(playerDataFolder, uuid + ".dat");
                   ServerConfigHandler.createDirectory(playerDataFolder);
-                  if (!file.renameTo(file2)) {
+                  if (!filex.renameTo(file2x)) {
                      throw new ServerConfigHandler.ServerConfigException("Could not convert file for " + fileName);
                   }
                }
@@ -446,3 +446,5 @@
       }
    }
 }
+
+// Fukkit Marker
