--- a/net/minecraft/server/ServerNetworkIo.java
+++ b/net/minecraft/server/ServerNetworkIo.java
@@ -155,8 +155,9 @@
 
                      LOGGER.warn("Failed to handle packet for {}", clientConnection.getAddress(), var7);
                      Text text = new LiteralText("Internal server error");
+                     ClientConnection $ = clientConnection;
                      clientConnection.send(new DisconnectS2CPacket(text), (future) -> {
-                        clientConnection.disconnect(text);
+                        $.disconnect(text);
                      });
                      clientConnection.disableAutoRead();
                   }
@@ -173,3 +174,5 @@
       return this.server;
    }
 }
+
+// Fukkit Marker
