--- a/net/minecraft/entity/projectile/ProjectileEntity.java
+++ b/net/minecraft/entity/projectile/ProjectileEntity.java
@@ -155,7 +155,7 @@
       this.yaw = updateRotation(this.prevYaw, (float)(MathHelper.atan2(vec3d.x, vec3d.z) * 57.2957763671875D));
    }
 
-   protected static float updateRotation(float f, float g) {
+   public static float updateRotation(float f, float g) {
       while(g - f < -180.0F) {
          f -= 360.0F;
       }
@@ -167,3 +167,5 @@
       return MathHelper.lerp(0.2F, f, g);
    }
 }
+
+// Fukkit Marker
