--- a/net/minecraft/world/chunk/ChunkStatus.java
+++ b/net/minecraft/world/chunk/ChunkStatus.java
@@ -297,3 +297,5 @@
       CompletableFuture<Either<Chunk, ChunkHolder.Unloaded>> doWork(ChunkStatus targetStatus, ServerWorld world, ChunkGenerator generator, StructureManager structureManager, ServerLightingProvider lightingProvider, Function<Chunk, CompletableFuture<Either<Chunk, ChunkHolder.Unloaded>>> function, List<Chunk> surroundingChunks, Chunk chunk);
    }
 }
+
+// Fukkit Marker
