--- a/net/minecraft/entity/ai/brain/task/GoToWorkTask.java
+++ b/net/minecraft/entity/ai/brain/task/GoToWorkTask.java
@@ -25,7 +25,7 @@
    protected void run(ServerWorld serverWorld, VillagerEntity villagerEntity, long l) {
       GlobalPos globalPos = (GlobalPos)villagerEntity.getBrain().getOptionalMemory(MemoryModuleType.POTENTIAL_JOB_SITE).get();
       villagerEntity.getBrain().forget(MemoryModuleType.POTENTIAL_JOB_SITE);
-      villagerEntity.getBrain().remember(MemoryModuleType.JOB_SITE, (Object)globalPos);
+      villagerEntity.getBrain().remember(MemoryModuleType.JOB_SITE, globalPos);
       serverWorld.sendEntityStatus(villagerEntity, (byte)14);
       if (villagerEntity.getVillagerData().getProfession() == VillagerProfession.NONE) {
          MinecraftServer minecraftServer = serverWorld.getServer();
@@ -42,3 +42,5 @@
       }
    }
 }
+
+// Fukkit Marker
