--- a/net/minecraft/entity/ai/brain/task/UpdateAttackTargetTask.java
+++ b/net/minecraft/entity/ai/brain/task/UpdateAttackTargetTask.java
@@ -34,7 +34,7 @@
    }
 
    protected void run(ServerWorld serverWorld, E mobEntity, long l) {
-      ((Optional)this.targetGetter.apply(mobEntity)).ifPresent((livingEntity) -> this.updateAttackTarget(mobEntity, livingEntity));
+      (this.targetGetter.apply(mobEntity)).ifPresent((livingEntity) -> this.updateAttackTarget(mobEntity, livingEntity));
    }
 
    private void updateAttackTarget(E entity, LivingEntity target) {
@@ -42,3 +42,5 @@
       entity.getBrain().forget(MemoryModuleType.CANT_REACH_WALK_TARGET_SINCE);
    }
 }
+
+// Fukkit Marker
