--- a/net/minecraft/entity/mob/CreeperEntity.java
+++ b/net/minecraft/entity/mob/CreeperEntity.java
@@ -208,7 +208,7 @@
          this.world.playSound(player, this.getX(), this.getY(), this.getZ(), SoundEvents.ITEM_FLINTANDSTEEL_USE, this.getSoundCategory(), 1.0F, this.random.nextFloat() * 0.4F + 0.8F);
          if (!this.world.isClient) {
             this.ignite();
-            itemStack.damage(1, (LivingEntity)player, (Consumer)((playerEntity) -> {
+            itemStack.damage(1, (LivingEntity)player, ((playerEntity) -> {
                playerEntity.sendToolBreakStatus(hand);
             }));
          }
@@ -274,3 +274,5 @@
       IGNITED = DataTracker.registerData(CreeperEntity.class, TrackedDataHandlerRegistry.BOOLEAN);
    }
 }
+
+// Fukkit Marker
