--- a/net/minecraft/entity/mob/MobEntity.java
+++ b/net/minecraft/entity/mob/MobEntity.java
@@ -1048,7 +1048,7 @@
          if (itemStack.getItem() instanceof SpawnEggItem) {
             if (this.world instanceof ServerWorld) {
                SpawnEggItem spawnEggItem = (SpawnEggItem)itemStack.getItem();
-               Optional<MobEntity> optional = spawnEggItem.spawnBaby(playerEntity, this, this.getType(), (ServerWorld)this.world, this.getPos(), itemStack);
+               Optional<MobEntity> optional = spawnEggItem.spawnBaby(playerEntity, this, (EntityType<? extends MobEntity>) this.getType(), (ServerWorld)this.world, this.getPos(), itemStack);
                optional.ifPresent((mobEntity) -> this.onPlayerSpawnedChild(playerEntity, mobEntity));
                return optional.isPresent() ? ActionResult.SUCCESS : ActionResult.PASS;
             } else {
@@ -1388,3 +1388,5 @@
       this.detachLeash(true, false);
    }
 }
+
+// Fukkit Marker
