--- a/net/minecraft/server/dedicated/AbstractPropertiesHandler.java
+++ b/net/minecraft/server/dedicated/AbstractPropertiesHandler.java
@@ -97,7 +97,7 @@
    private static <V extends Number> Function<String, V> wrapNumberParser(Function<String, V> parser) {
       return (string) -> {
          try {
-            return (Number)parser.apply(string);
+            return parser.apply(string);
          } catch (NumberFormatException var3) {
             return null;
          }
@@ -141,7 +141,7 @@
       String string = this.getStringValue(key);
       V object = (V)MoreObjects.firstNonNull(string != null ? parser.apply(string) : null, fallback);
       this.properties.put(key, stringifier.apply(object));
-      return new AbstractPropertiesHandler.PropertyAccessor<>(key, object, stringifier);
+      return new AbstractPropertiesHandler<T>.PropertyAccessor<>(key, object, stringifier);
    }
 
    protected <V> V get(String key, Function<String, V> parser, UnaryOperator<V> parsedTransformer, Function<V, String> stringifier, V fallback) {
@@ -238,3 +238,5 @@
       }
    }
 }
+
+// Fukkit Marker
