--- a/net/minecraft/entity/player/PlayerInventory.java
+++ b/net/minecraft/entity/player/PlayerInventory.java
@@ -302,8 +302,8 @@
          } catch (Throwable var6) {
             CrashReport crashReport = CrashReport.create(var6, "Adding item to inventory");
             CrashReportSection crashReportSection = crashReport.addElement("Item being added");
-            crashReportSection.add("Item ID", (Object)Item.getRawId(stack.getItem()));
-            crashReportSection.add("Item data", (Object)stack.getDamage());
+            crashReportSection.add("Item ID", Item.getRawId(stack.getItem()));
+            crashReportSection.add("Item data", stack.getDamage());
             crashReportSection.add("Item name", () -> {
                return stack.getName().getString();
             });
@@ -539,7 +539,7 @@
          for(int i = 0; i < this.armor.size(); ++i) {
             ItemStack itemStack = (ItemStack)this.armor.get(i);
             if ((!damageSource.isFire() || !itemStack.getItem().isFireproof()) && itemStack.getItem() instanceof ArmorItem) {
-               itemStack.damage((int)f, (LivingEntity)this.player, (Consumer)((playerEntity) -> {
+               itemStack.damage((int)f, (LivingEntity)this.player, ((playerEntity) -> {
                   playerEntity.sendEquipmentBreakStatus(EquipmentSlot.fromTypeIndex(EquipmentSlot.Type.ARMOR, i));
                }));
             }
@@ -655,3 +655,5 @@
 
    }
 }
+
+// Fukkit Marker
