--- a/net/minecraft/entity/passive/BeeEntity.java
+++ b/net/minecraft/entity/passive/BeeEntity.java
@@ -587,7 +587,7 @@
 
    class EnterHiveGoal extends BeeEntity.NotAngryGoal {
       private EnterHiveGoal() {
-         super(null);
+         super();
       }
 
       public boolean canBeeStart() {
@@ -636,7 +636,7 @@
 
    class GrowCropsGoal extends BeeEntity.NotAngryGoal {
       private GrowCropsGoal() {
-         super(null);
+         super();
       }
 
       public boolean canBeeStart() {
@@ -699,7 +699,7 @@
 
    class FindHiveGoal extends BeeEntity.NotAngryGoal {
       private FindHiveGoal() {
-         super(null);
+         super();
       }
 
       public boolean canBeeStart() {
@@ -764,7 +764,7 @@
       private int ticks = 0;
 
       PollinateGoal() {
-         super(null);
+         super();
          this.setControls(EnumSet.of(Goal.Control.MOVE));
       }
 
@@ -935,7 +935,7 @@
       private int ticks;
 
       MoveToFlowerGoal() {
-         super(null);
+         super();
          this.ticks = BeeEntity.this.world.random.nextInt(10);
          this.setControls(EnumSet.of(Goal.Control.MOVE));
       }
@@ -987,7 +987,7 @@
       private int ticksUntilLost;
 
       MoveToHiveGoal() {
-         super(null);
+         super();
          this.ticks = BeeEntity.this.world.random.nextInt(10);
          this.possibleHives = Lists.newArrayList();
          this.path = null;
@@ -1123,7 +1123,6 @@
             vec3d3 = BeeEntity.this.getRotationVec(0.0F);
          }
 
-         int i = true;
          Vec3d vec3d4 = TargetFinder.findAirTarget(BeeEntity.this, 8, 7, vec3d3, 1.5707964F, 2, 1);
          return vec3d4 != null ? vec3d4 : TargetFinder.findGroundTarget(BeeEntity.this, 8, 4, -2, vec3d3, 1.5707963705062866D);
       }
@@ -1188,3 +1187,5 @@
       }
    }
 }
+
+// Fukkit Marker
