--- a/net/minecraft/entity/passive/CatEntity.java
+++ b/net/minecraft/entity/passive/CatEntity.java
@@ -601,8 +601,7 @@
       private final CatEntity cat;
 
       public CatFleeGoal(CatEntity cat, Class<T> fleeFromType, float distance, double slowSpeed, double fastSpeed) {
-         Predicate var10006 = EntityPredicates.EXCEPT_CREATIVE_OR_SPECTATOR;
-         super(cat, fleeFromType, distance, slowSpeed, fastSpeed, var10006::test);
+         super(cat, fleeFromType, distance, slowSpeed, fastSpeed, EntityPredicates.EXCEPT_CREATIVE_OR_SPECTATOR::test);
          this.cat = cat;
       }
 
@@ -615,3 +614,5 @@
       }
    }
 }
+
+// Fukkit Marker
