--- a/net/minecraft/entity/passive/FoxEntity.java
+++ b/net/minecraft/entity/passive/FoxEntity.java
@@ -316,13 +316,13 @@
 
    private List<UUID> getTrustedUuids() {
       List<UUID> list = Lists.newArrayList();
-      list.add(((Optional)this.dataTracker.get(OWNER)).orElse((Object)null));
-      list.add(((Optional)this.dataTracker.get(OTHER_TRUSTED)).orElse((Object)null));
+      list.add((this.dataTracker.get(OWNER)).orElse(null));
+      list.add((this.dataTracker.get(OTHER_TRUSTED)).orElse(null));
       return list;
    }
 
    private void addTrustedUuid(@Nullable UUID uuid) {
-      if (((Optional)this.dataTracker.get(OWNER)).isPresent()) {
+      if ((this.dataTracker.get(OWNER)).isPresent()) {
          this.dataTracker.set(OTHER_TRUSTED, Optional.ofNullable(uuid));
       } else {
          this.dataTracker.set(OWNER, Optional.ofNullable(uuid));
@@ -629,7 +629,6 @@
       double d = chasedEntity.getZ() - fox.getZ();
       double e = chasedEntity.getX() - fox.getX();
       double f = d / e;
-      int i = true;
 
       for(int j = 0; j < 6; ++j) {
          double g = f == 0.0D ? 0.0D : d * (double)((float)j / 6.0F);
@@ -1394,3 +1393,5 @@
       }
    }
 }
+
+// Fukkit Marker
