--- a/net/minecraft/entity/passive/IronGolemEntity.java
+++ b/net/minecraft/entity/passive/IronGolemEntity.java
@@ -75,7 +75,7 @@
       this.goalSelector.add(8, new LookAroundGoal(this));
       this.targetSelector.add(1, new TrackIronGolemTargetGoal(this));
       this.targetSelector.add(2, new RevengeGoal(this));
-      this.targetSelector.add(3, new FollowTargetGoal(this, PlayerEntity.class, 10, true, false, this::shouldAngerAt));
+      this.targetSelector.add(3, new FollowTargetGoal<>(this, PlayerEntity.class, 10, true, false, this::shouldAngerAt));
       this.targetSelector.add(3, new FollowTargetGoal(this, MobEntity.class, 5, false, false, (livingEntity) -> livingEntity instanceof Monster && !(livingEntity instanceof CreeperEntity)));
       this.targetSelector.add(4, new UniversalAngerGoal<>(this, false));
    }
@@ -306,3 +306,5 @@
       }
    }
 }
+
+// Fukkit Marker
