--- a/net/minecraft/entity/passive/SheepEntity.java
+++ b/net/minecraft/entity/passive/SheepEntity.java
@@ -38,6 +38,7 @@
 import net.minecraft.entity.player.PlayerEntity;
 import net.minecraft.inventory.CraftingInventory;
 import net.minecraft.item.DyeItem;
+import net.minecraft.item.Item;
 import net.minecraft.item.ItemConvertible;
 import net.minecraft.item.ItemStack;
 import net.minecraft.item.Items;
@@ -202,7 +203,7 @@
       if (itemStack.getItem() == Items.SHEARS) {
          if (!this.world.isClient && this.isShearable()) {
             this.sheared(SoundCategory.PLAYERS);
-            itemStack.damage(1, (LivingEntity)player, (Consumer)((playerEntity) -> {
+            itemStack.damage(1, (LivingEntity)player, ((playerEntity) -> {
                playerEntity.sendToolBreakStatus(hand);
             }));
             return ActionResult.SUCCESS;
@@ -323,7 +324,7 @@
       DyeColor dyeColor = ((SheepEntity)firstParent).getColor();
       DyeColor dyeColor2 = ((SheepEntity)secondParent).getColor();
       CraftingInventory craftingInventory = createDyeMixingCraftingInventory(dyeColor, dyeColor2);
-      Optional var10000 = this.world.getRecipeManager().getFirstMatch(RecipeType.CRAFTING, craftingInventory, this.world).map((craftingRecipe) -> {
+      Optional<Item> var10000 = this.world.getRecipeManager().getFirstMatch(RecipeType.CRAFTING, craftingInventory, this.world).map((craftingRecipe) -> {
          return craftingRecipe.craft(craftingInventory);
       }).map(ItemStack::getItem);
       DyeItem.class.getClass();
@@ -374,3 +375,5 @@
       }, SheepEntity::getDyedColor)));
    }
 }
+
+// Fukkit Marker
