--- a/net/minecraft/screen/AbstractFurnaceScreenHandler.java
+++ b/net/minecraft/screen/AbstractFurnaceScreenHandler.java
@@ -68,7 +68,7 @@
    }
 
    public void fillInputSlots(boolean craftAll, Recipe<?> recipe, ServerPlayerEntity player) {
-      (new FurnaceInputSlotFiller<>(this)).fillInputSlots(player, recipe, craftAll);
+      (new FurnaceInputSlotFiller<>(this)).fillInputSlots(player, (Recipe) recipe, craftAll);
    }
 
    public boolean matches(Recipe<? super Inventory> recipe) {
@@ -147,3 +147,5 @@
       return AbstractFurnaceBlockEntity.canUseAsFuel(itemStack);
    }
 }
+
+// Fukkit Marker
