--- a/net/minecraft/screen/HorseScreenHandler.java
+++ b/net/minecraft/screen/HorseScreenHandler.java
@@ -17,9 +17,7 @@
       super((ScreenHandlerType)null, syncId);
       this.inventory = inventory;
       this.entity = entity;
-      int i = true;
       inventory.onOpen(playerInventory.player);
-      int j = true;
       this.addSlot(new Slot(inventory, 0, 8, 18) {
          public boolean canInsert(ItemStack stack) {
             return stack.getItem() == Items.SADDLE && !this.hasStack() && entity.canBeSaddled();
@@ -112,3 +110,5 @@
       this.inventory.onClose(player);
    }
 }
+
+// Fukkit Marker
