--- a/net/minecraft/screen/ScreenHandlerContext.java
+++ b/net/minecraft/screen/ScreenHandlerContext.java
@@ -38,9 +38,11 @@
    }
 
    default void run(BiConsumer<World, BlockPos> function) {
-      this.run((BiFunction)((world, blockPos) -> {
+      this.run(((world, blockPos) -> {
          function.accept(world, blockPos);
          return Optional.empty();
       }));
    }
 }
+
+// Fukkit Marker
