--- a/net/minecraft/entity/Entity.java
+++ b/net/minecraft/entity/Entity.java
@@ -2041,7 +2041,7 @@
                }
 
                return AreaHelper.method_30484(destination, arg, axis, vec3d, this.getDimensions(this.getPose()), this.getVelocity(), this.yaw, this.pitch);
-            }).orElse((Object)null);
+            }).orElse(null);
          }
       } else {
          BlockPos blockPos;
@@ -2641,3 +2641,5 @@
       void accept(Entity entity, double x, double y, double z);
    }
 }
+
+// Fukkit Marker
