--- a/net/minecraft/entity/LivingEntity.java
+++ b/net/minecraft/entity/LivingEntity.java
@@ -442,7 +442,7 @@
             entityAttributeInstance.addTemporaryModifier(new EntityAttributeModifier(SOUL_SPEED_BOOST_ID, "Soul speed boost", (double)(0.03F * (1.0F + (float)i * 0.35F)), EntityAttributeModifier.Operation.ADDITION));
             if (this.getRandom().nextFloat() < 0.04F) {
                ItemStack itemStack = this.getEquippedStack(EquipmentSlot.FEET);
-               itemStack.damage(1, (LivingEntity)this, (Consumer)((livingEntity) -> {
+               itemStack.damage(1, (LivingEntity)this, ((livingEntity) -> {
                   livingEntity.sendEquipmentBreakStatus(EquipmentSlot.FEET);
                }));
             }
@@ -2500,7 +2500,7 @@
          if (itemStack.getItem() == Items.ELYTRA && ElytraItem.isUsable(itemStack)) {
             bl = true;
             if (!this.world.isClient && (this.roll + 1) % 20 == 0) {
-               itemStack.damage(1, (LivingEntity)this, (Consumer)((livingEntity) -> {
+               itemStack.damage(1, (LivingEntity)this, ((livingEntity) -> {
                   livingEntity.sendEquipmentBreakStatus(EquipmentSlot.CHEST);
                }));
             }
@@ -2767,7 +2767,6 @@
 
    protected void setLivingFlag(int mask, boolean value) {
       int i = (Byte)this.dataTracker.get(LIVING_FLAGS);
-      int i;
       if (value) {
          i = i | mask;
       } else {
@@ -3010,7 +3009,7 @@
    }
 
    public Optional<BlockPos> getSleepingPosition() {
-      return (Optional)this.dataTracker.get(SLEEPING_POSITION);
+      return this.dataTracker.get(SLEEPING_POSITION);
    }
 
    public void setSleepingPosition(BlockPos pos) {
@@ -3053,7 +3052,7 @@
    }
 
    public void wakeUp() {
-      Optional var10000 = this.getSleepingPosition();
+      Optional<BlockPos> var10000 = this.getSleepingPosition();
       World var10001 = this.world;
       var10001.getClass();
       var10000.filter(var10001::isChunkLoaded).ifPresent((blockPos) -> {
@@ -3081,7 +3080,7 @@
    @Nullable
    @Environment(EnvType.CLIENT)
    public Direction getSleepingDirection() {
-      BlockPos blockPos = (BlockPos)this.getSleepingPosition().orElse((Object)null);
+      BlockPos blockPos = (BlockPos)this.getSleepingPosition().orElse(null);
       return blockPos != null ? BedBlock.getDirection(this.world, blockPos) : null;
    }
 
@@ -3178,3 +3177,5 @@
       SLEEPING_DIMENSIONS = EntityDimensions.fixed(0.2F, 0.2F);
    }
 }
+
+// Fukkit Marker
