--- a/net/minecraft/server/network/ServerPlayNetworkHandler.java
+++ b/net/minecraft/server/network/ServerPlayNetworkHandler.java
@@ -309,7 +309,7 @@
       };
       TextStream textStream = this.player.getTextStream();
       if (textStream != null) {
-         ((CompletableFuture)backingFilterer.apply(textStream, text)).thenAcceptAsync((optional) -> optional.ifPresent(consumer2), threadExecutor);
+         (backingFilterer.apply(textStream, text)).thenAcceptAsync((optional) -> optional.ifPresent(consumer2), threadExecutor);
       } else {
          threadExecutor.execute(() -> consumer2.accept(text));
       }
@@ -1404,3 +1404,5 @@
       }
    }
 }
+
+// Fukkit Marker
