--- a/net/minecraft/village/raid/Raid.java
+++ b/net/minecraft/village/raid/Raid.java
@@ -400,7 +400,7 @@
       Set<RaiderEntity> set = Sets.newHashSet();
 
       while(iterator.hasNext()) {
-         for(RaiderEntity raiderEntity : (Set)iterator.next()) {
+         for(RaiderEntity raiderEntity : iterator.next()) {
             BlockPos blockPos = raiderEntity.getBlockPos();
             if (!raiderEntity.removed && raiderEntity.world.getRegistryKey() == this.world.getRegistryKey() && !(this.center.getSquaredDistance(blockPos) >= 12544.0D)) {
                if (raiderEntity.age > 600) {
@@ -518,10 +518,10 @@
 
    public float getCurrentRaiderHealth() {
       float f = 0.0F;
-      Iterator var2 = this.waveToRaiders.values().iterator();
+      Iterator<Set<RaiderEntity>> var2 = this.waveToRaiders.values().iterator();
 
       while(var2.hasNext()) {
-         for(RaiderEntity raiderEntity : (Set)var2.next()) {
+         for(RaiderEntity raiderEntity : var2.next()) {
             f += raiderEntity.getHealth();
          }
       }
@@ -781,3 +781,5 @@
       }
    }
 }
+
+// Fukkit Marker
