--- a/net/minecraft/server/command/CommandManager.java
+++ b/net/minecraft/server/command/CommandManager.java
@@ -205,7 +205,7 @@
    private void makeTreeForSource(CommandNode<ServerCommandSource> tree, CommandNode<CommandSource> result, ServerCommandSource source, Map<CommandNode<ServerCommandSource>, CommandNode<CommandSource>> resultNodes) {
       for(CommandNode<ServerCommandSource> commandNode : tree.getChildren()) {
          if (commandNode.canUse(source)) {
-            ArgumentBuilder<CommandSource, ?> argumentBuilder = commandNode.createBuilder();
+            ArgumentBuilder<CommandSource, ?> argumentBuilder = (ArgumentBuilder<CommandSource, ?>) (Object) commandNode.createBuilder();
             argumentBuilder.requires((commandSource) -> true);
             if (argumentBuilder.getCommand() != null) {
                argumentBuilder.executes((commandContext) -> 0);
@@ -299,3 +299,5 @@
       }
    }
 }
+
+// Fukkit Marker
