--- a/net/minecraft/server/command/SpreadPlayersCommand.java
+++ b/net/minecraft/server/command/SpreadPlayersCommand.java
@@ -58,7 +58,7 @@
          if (entity instanceof PlayerEntity) {
             set.add(entity.getScoreboardTeam());
          } else {
-            set.add((Object)null);
+            set.add(null);
          }
       }
 
@@ -258,3 +258,5 @@
       }
    }
 }
+
+// Fukkit Marker
