--- a/net/minecraft/world/ChunkSerializer.java
+++ b/net/minecraft/world/ChunkSerializer.java
@@ -68,10 +68,10 @@
 
       BiomeArray biomeArray = new BiomeArray(world.getRegistryManager().get(Registry.BIOME_KEY), pos, biomeSource, compoundTag.contains("Biomes", 11) ? compoundTag.getIntArray("Biomes") : null);
       UpgradeData upgradeData = compoundTag.contains("UpgradeData", 10) ? new UpgradeData(compoundTag.getCompound("UpgradeData")) : UpgradeData.NO_UPGRADE_DATA;
-      ChunkTickScheduler<Block> chunkTickScheduler = new ChunkTickScheduler((block) -> {
+      ChunkTickScheduler<Block> chunkTickScheduler = new ChunkTickScheduler<>((block) -> {
          return block == null || block.getDefaultState().isAir();
       }, pos, compoundTag.getList("ToBeTicked", 9));
-      ChunkTickScheduler<Fluid> chunkTickScheduler2 = new ChunkTickScheduler((fluid) -> {
+      ChunkTickScheduler<Fluid> chunkTickScheduler2 = new ChunkTickScheduler<>((fluid) -> {
          return fluid == null || fluid == Fluids.EMPTY;
       }, pos, compoundTag.getList("LiquidsToBeTicked", 9));
       boolean bl = compoundTag.getBoolean("isLightOn");
@@ -512,3 +512,5 @@
       return listTag;
    }
 }
+
+// Fukkit Marker
