--- a/net/minecraft/world/GameRules.java
+++ b/net/minecraft/world/GameRules.java
@@ -138,7 +138,7 @@
    }
 
    public <T extends GameRules.Rule<T>> T get(GameRules.Key<T> key) {
-      return (GameRules.Rule)this.rules.get(key);
+      return (T) this.rules.get(key);
    }
 
    public CompoundTag toNbt() {
@@ -255,7 +255,7 @@
       private boolean value;
 
       private static GameRules.Type<GameRules.BooleanRule> create(boolean initialValue, BiConsumer<MinecraftServer, GameRules.BooleanRule> changeCallback) {
-         return new GameRules.Type(BoolArgumentType::bool, (type) -> {
+         return new GameRules.Type<>(BoolArgumentType::bool, (type) -> {
             return new GameRules.BooleanRule(type, initialValue);
          }, changeCallback, GameRules.Visitor::visitBoolean);
       }
@@ -314,7 +314,7 @@
       private int value;
 
       private static GameRules.Type<GameRules.IntRule> create(int initialValue, BiConsumer<MinecraftServer, GameRules.IntRule> changeCallback) {
-         return new GameRules.Type(IntegerArgumentType::integer, (type) -> {
+         return new GameRules.Type<>(IntegerArgumentType::integer, (type) -> {
             return new GameRules.IntRule(type, initialValue);
          }, changeCallback, GameRules.Visitor::visitInt);
       }
@@ -446,7 +446,7 @@
       }
 
       public T createRule() {
-         return (GameRules.Rule)this.ruleFactory.apply(this);
+         return (T) this.ruleFactory.apply(this);
       }
 
       public void accept(GameRules.Visitor consumer, GameRules.Key<T> key) {
@@ -547,3 +547,5 @@
       }
    }
 }
+
+// Fukkit Marker
