--- a/net/minecraft/world/GameRules.java
+++ b/net/minecraft/world/GameRules.java
@@ -170,8 +170,8 @@
    }
 
    private static <T extends GameRules.Rule<T>> void accept(GameRules.Visitor consumer, GameRules.Key<?> key, GameRules.Type<?> type) {
-      consumer.visit(key, type);
-      type.accept(consumer, key);
+      consumer.visit((Key) key, type);
+      type.accept(consumer, (Key) key);
    }
 
    public boolean getBoolean(GameRules.Key<GameRules.BooleanRule> rule) {
@@ -436,3 +436,5 @@
       }
    }
 }
+
+// Fukkit Marker
