--- a/net/minecraft/world/PortalForcer.java
+++ b/net/minecraft/world/PortalForcer.java
@@ -29,12 +29,12 @@
       PointOfInterestStorage pointOfInterestStorage = this.world.getPointOfInterestStorage();
       int i = bl ? 16 : 128;
       pointOfInterestStorage.preloadChunks(this.world, blockPos, i);
-      Optional<PointOfInterest> optional = pointOfInterestStorage.getInSquare((pointOfInterestType) -> pointOfInterestType == PointOfInterestType.NETHER_PORTAL, blockPos, i, PointOfInterestStorage.OccupationStatus.ANY).sorted(Comparator.comparingDouble((pointOfInterest) -> pointOfInterest.getPos().getSquaredDistance(blockPos)).thenComparingInt((pointOfInterest) -> pointOfInterest.getPos().getY())).filter((pointOfInterest) -> this.world.getBlockState(pointOfInterest.getPos()).contains(Properties.HORIZONTAL_AXIS)).findFirst();
+      Optional<PointOfInterest> optional = pointOfInterestStorage.getInSquare((pointOfInterestType) -> pointOfInterestType == PointOfInterestType.NETHER_PORTAL, blockPos, i, PointOfInterestStorage.OccupationStatus.ANY).sorted(Comparator.comparingDouble((PointOfInterest pointOfInterest) -> pointOfInterest.getPos().getSquaredDistance(blockPos)).thenComparingInt((pointOfInterest) -> pointOfInterest.getPos().getY())).filter((pointOfInterest) -> this.world.getBlockState(pointOfInterest.getPos()).contains(Properties.HORIZONTAL_AXIS)).findFirst();
       return optional.map((pointOfInterest) -> {
-         BlockPos blockPos = pointOfInterest.getPos();
-         this.world.getChunkManager().addTicket(ChunkTicketType.PORTAL, new ChunkPos(blockPos), 3, blockPos);
-         BlockState blockState = this.world.getBlockState(blockPos);
-         return class_5459.method_30574(blockPos, blockState.get(Properties.HORIZONTAL_AXIS), 21, Direction.Axis.Y, 21, (blockPosx) -> this.world.getBlockState(blockPosx) == blockState);
+         BlockPos blockPos2 = pointOfInterest.getPos();
+         this.world.getChunkManager().addTicket(ChunkTicketType.PORTAL, new ChunkPos(blockPos2), 3, blockPos2);
+         BlockState blockState = this.world.getBlockState(blockPos2);
+         return class_5459.method_30574(blockPos2, blockState.get(Properties.HORIZONTAL_AXIS), 21, Direction.Axis.Y, 21, (blockPosx) -> this.world.getBlockState(blockPosx) == blockState);
       });
    }
 
@@ -147,3 +147,5 @@
       return true;
    }
 }
+
+// Fukkit Marker
