--- a/net/minecraft/world/SpawnHelper.java
+++ b/net/minecraft/world/SpawnHelper.java
@@ -77,10 +77,11 @@
          if (spawnGroup != SpawnGroup.MISC) {
             BlockPos blockPos = entity.getBlockPos();
             long l = ChunkPos.toLong(blockPos.getX() >> 4, blockPos.getZ() >> 4);
+            Entity finalEntity = entity;
             chunkSource.query(l, (worldChunk) -> {
-               SpawnSettings.SpawnDensity spawnDensity = getBiomeDirectly(blockPos, worldChunk).getSpawnSettings().getSpawnDensity(entity.getType());
+               SpawnSettings.SpawnDensity spawnDensity = getBiomeDirectly(blockPos, worldChunk).getSpawnSettings().getSpawnDensity(finalEntity.getType());
                if (spawnDensity != null) {
-                  gravityField.addPoint(entity.getBlockPos(), spawnDensity.getMass());
+                  gravityField.addPoint(finalEntity.getBlockPos(), spawnDensity.getMass());
                }
 
                object2IntOpenHashMap.addTo(spawnGroup, 1);
@@ -476,3 +477,5 @@
       void run(MobEntity entity, Chunk chunk);
    }
 }
+
+// Fukkit Marker
