--- a/net/minecraft/server/world/ChunkTicketManager.java
+++ b/net/minecraft/server/world/ChunkTicketManager.java
@@ -63,7 +63,7 @@
 
       while(objectIterator.hasNext()) {
          Entry<SortedArraySet<ChunkTicket<?>>> entry = (Entry)objectIterator.next();
-         if (((SortedArraySet)entry.getValue()).removeIf((chunkTicket) -> chunkTicket.isExpired(this.age))) {
+         if ((entry.getValue()).removeIf((chunkTicket) -> chunkTicket.isExpired(this.age))) {
             this.distanceFromTicketTracker.updateLevel(entry.getLongKey(), getLevel((SortedArraySet)entry.getValue()), false);
          }
 
@@ -392,3 +392,5 @@
       }
    }
 }
+
+// Fukkit Marker
