--- a/net/minecraft/server/world/ServerChunkManager.java
+++ b/net/minecraft/server/world/ServerChunkManager.java
@@ -155,11 +155,11 @@
          if (chunkHolder == null) {
             return null;
          } else {
-            Either<Chunk, ChunkHolder.Unloaded> either = (Either)chunkHolder.getValidFutureFor(ChunkStatus.FULL).getNow((Object)null);
+            Either<Chunk, ChunkHolder.Unloaded> either = (Either)chunkHolder.getValidFutureFor(ChunkStatus.FULL).getNow(null);
             if (either == null) {
                return null;
             } else {
-               Chunk chunk2 = (Chunk)either.left().orElse((Object)null);
+               Chunk chunk2 = (Chunk)either.left().orElse(null);
                if (chunk2 != null) {
                   this.putInCache(l, chunk2, ChunkStatus.FULL);
                   if (chunk2 instanceof WorldChunk) {
@@ -497,3 +497,5 @@
       }
    }
 }
+
+// Fukkit Marker
