--- a/net/minecraft/fluid/FlowableFluid.java
+++ b/net/minecraft/fluid/FlowableFluid.java
@@ -433,3 +433,5 @@
       return state.getLevel() == 9 && isFluidAboveEqual(state, world, pos) ? VoxelShapes.fullCube() : (VoxelShape)this.shapeCache.computeIfAbsent(state, (fluidState) -> VoxelShapes.cuboid(0.0D, 0.0D, 0.0D, 1.0D, (double)fluidState.getHeight(world, pos), 1.0D));
    }
 }
+
+// Fukkit Marker
