--- a/net/minecraft/command/argument/BlockArgumentParser.java
+++ b/net/minecraft/command/argument/BlockArgumentParser.java
@@ -199,7 +199,7 @@
    private static <T extends Comparable<T>> SuggestionsBuilder suggestPropertyValues(SuggestionsBuilder suggestionsBuilder, Property<T> property) {
       for(T comparable : property.getValues()) {
          if (comparable instanceof Integer) {
-            suggestionsBuilder.suggest(comparable);
+            suggestionsBuilder.suggest((Integer) comparable);
          } else {
             suggestionsBuilder.suggest(property.name(comparable));
          }
@@ -448,7 +448,7 @@
    private <T extends Comparable<T>> void parsePropertyValue(Property<T> property, String string, int i) throws CommandSyntaxException {
       Optional<T> optional = property.parse(string);
       if (optional.isPresent()) {
-         this.blockState = this.blockState.with(property, (Comparable)optional.get());
+         this.blockState = this.blockState.with(property, optional.get());
          this.blockProperties.put(property, optional.get());
       } else {
          this.reader.setCursor(i);
@@ -491,3 +491,5 @@
       return this.tagProperties;
    }
 }
+
+// Fukkit Marker
