--- a/net/minecraft/block/sapling/LargeTreeSaplingGenerator.java
+++ b/net/minecraft/block/sapling/LargeTreeSaplingGenerator.java
@@ -56,3 +56,5 @@
       return block == world.getBlockState(pos.add(x, 0, z)).getBlock() && block == world.getBlockState(pos.add(x + 1, 0, z)).getBlock() && block == world.getBlockState(pos.add(x, 0, z + 1)).getBlock() && block == world.getBlockState(pos.add(x + 1, 0, z + 1)).getBlock();
    }
 }
+
+// Fukkit Marker
