--- a/net/minecraft/entity/boss/WitherEntity.java
+++ b/net/minecraft/entity/boss/WitherEntity.java
@@ -4,11 +4,7 @@
 import java.util.EnumSet;
 import java.util.List;
 import java.util.function.Predicate;
-import net.fabricmc.api.EnvType;
-import net.fabricmc.api.EnvironmentInterface;
-import net.fabricmc.api.EnvironmentInterfaces;
 import net.minecraft.block.BlockState;
-import net.minecraft.client.render.entity.feature.SkinOverlayOwner;
 import net.minecraft.entity.Entity;
 import net.minecraft.entity.EntityGroup;
 import net.minecraft.entity.EntityType;
@@ -53,10 +49,6 @@
 import net.minecraft.world.explosion.Explosion;
 import org.jetbrains.annotations.Nullable;
 
-@EnvironmentInterfaces({@EnvironmentInterface(
-   value = EnvType.CLIENT,
-   itf = SkinOverlayOwner.class
-)})
 public class WitherEntity extends HostileEntity implements RangedAttackMob {
    private static final TrackedData<Integer> TRACKED_ENTITY_ID_1 = DataTracker.registerData(WitherEntity.class, TrackedDataHandlerRegistry.INTEGER);
    private static final TrackedData<Integer> TRACKED_ENTITY_ID_2 = DataTracker.registerData(WitherEntity.class, TrackedDataHandlerRegistry.INTEGER);
@@ -482,7 +474,7 @@
    }
 
    public int getTrackedEntityId(int headIndex) {
-      return this.dataTracker.get((TrackedData)TRACKED_ENTITY_IDS.get(headIndex));
+      return this.dataTracker.get(TRACKED_ENTITY_IDS.get(headIndex));
    }
 
    public void setTrackedEntityId(int headIndex, int id) {
@@ -519,3 +511,5 @@
       }
    }
 }
+
+// Fukkit Marker
