--- a/net/minecraft/block/BeehiveBlock.java
+++ b/net/minecraft/block/BeehiveBlock.java
@@ -113,7 +113,7 @@
          if (itemStack.getItem() == Items.SHEARS) {
             world.playSound(player, player.getX(), player.getY(), player.getZ(), SoundEvents.BLOCK_BEEHIVE_SHEAR, SoundCategory.NEUTRAL, 1.0F, 1.0F);
             dropHoneycomb(world, pos);
-            itemStack.damage(1, (LivingEntity)player, (Consumer)((playerx) -> {
+            itemStack.damage(1, (LivingEntity)player, ((playerx) -> {
                playerx.sendToolBreakStatus(hand);
             }));
             bl = true;
@@ -297,3 +297,5 @@
       HONEY_LEVEL = Properties.HONEY_LEVEL;
    }
 }
+
+// Fukkit Marker
