--- a/net/minecraft/world/updater/WorldUpdater.java
+++ b/net/minecraft/world/updater/WorldUpdater.java
@@ -95,10 +95,9 @@
          float f = (float)this.totalChunkCount;
          ImmutableMap<RegistryKey<World>, ListIterator<ChunkPos>> immutableMap = builder.build();
          Builder<RegistryKey<World>, VersionedChunkStorage> builder2 = ImmutableMap.builder();
-         UnmodifiableIterator immutableMap2 = this.worlds.iterator();
 
-         while(immutableMap2.hasNext()) {
-            RegistryKey<World> registryKey2 = (RegistryKey)immutableMap2.next();
+         for (RegistryKey<World> world : this.worlds) {
+            RegistryKey<World> registryKey2 = (RegistryKey) world;
             File file = this.session.getWorldDirectory(registryKey2);
             builder2.put(registryKey2, new VersionedChunkStorage(new File(file, "region"), this.dataFixer, true));
          }
@@ -247,3 +246,5 @@
       return this.status;
    }
 }
+
+// Fukkit Marker
