--- a/net/minecraft/item/FlintAndSteelItem.java
+++ b/net/minecraft/item/FlintAndSteelItem.java
@@ -29,7 +29,7 @@
          world.playSound(playerEntity, blockPos, SoundEvents.ITEM_FLINTANDSTEEL_USE, SoundCategory.BLOCKS, 1.0F, RANDOM.nextFloat() * 0.4F + 0.8F);
          world.setBlockState(blockPos, (BlockState)blockState.with(Properties.LIT, true), 11);
          if (playerEntity != null) {
-            context.getStack().damage(1, (LivingEntity)playerEntity, (Consumer)((p) -> {
+            context.getStack().damage(1, (LivingEntity)playerEntity, ((p) -> {
                p.sendToolBreakStatus(context.getHand());
             }));
          }
@@ -44,7 +44,7 @@
             ItemStack itemStack = context.getStack();
             if (playerEntity instanceof ServerPlayerEntity) {
                Criteria.PLACED_BLOCK.trigger((ServerPlayerEntity)playerEntity, blockPos2, itemStack);
-               itemStack.damage(1, (LivingEntity)playerEntity, (Consumer)((p) -> {
+               itemStack.damage(1, (LivingEntity)playerEntity, ((p) -> {
                   p.sendToolBreakStatus(context.getHand());
                }));
             }
@@ -56,3 +56,5 @@
       }
    }
 }
+
+// Fukkit Marker
