--- a/net/minecraft/block/entity/AbstractFurnaceBlockEntity.java
+++ b/net/minecraft/block/entity/AbstractFurnaceBlockEntity.java
@@ -224,7 +224,7 @@
       if (!this.world.isClient) {
          ItemStack itemStack = this.inventory.get(1);
          if (this.isBurning() || !itemStack.isEmpty() && !this.inventory.get(0).isEmpty()) {
-            Recipe<?> recipe = (Recipe)this.world.getRecipeManager().getFirstMatch(this.recipeType, this, this.world).orElse((Object)null);
+            Recipe<?> recipe = (Recipe)this.world.getRecipeManager().getFirstMatch(this.recipeType, this, this.world).orElse(null);
             if (!this.isBurning() && this.canAcceptRecipeOutput(recipe)) {
                this.burnTime = this.getFuelTime(itemStack);
                this.fuelTime = this.burnTime;
@@ -477,3 +477,5 @@
 
    }
 }
+
+// Fukkit Marker
