--- a/net/minecraft/advancement/PlayerAdvancementTracker.java
+++ b/net/minecraft/advancement/PlayerAdvancementTracker.java
@@ -152,7 +152,7 @@
 
                Stream<Entry<Identifier, AdvancementProgress>> stream = map.entrySet().stream().sorted(Comparator.comparing(Entry::getValue));
 
-               for(Entry<Identifier, AdvancementProgress> entry : (List)stream.collect(Collectors.toList())) {
+               for(Entry<Identifier, AdvancementProgress> entry : stream.collect(Collectors.toList())) {
                   Advancement advancement = advancementLoader.get((Identifier)entry.getKey());
                   if (advancement == null) {
                      LOGGER.warn("Ignored advancement '{}' in progress file {} - it doesn't exist anymore?", entry.getKey(), this.advancementFile);
@@ -455,3 +455,5 @@
       }
    }
 }
+
+// Fukkit Marker
