--- a/net/minecraft/world/level/storage/LevelStorage.java
+++ b/net/minecraft/world/level/storage/LevelStorage.java
@@ -77,13 +77,11 @@
 
    private static <T> Pair<GeneratorOptions, Lifecycle> readGeneratorProperties(Dynamic<T> dynamic, DataFixer dataFixer, int i) {
       Dynamic<T> dynamic2 = dynamic.get("WorldGenSettings").orElseEmptyMap();
-      UnmodifiableIterator dynamic3 = GENERATOR_OPTION_KEYS.iterator();
 
-      while(dynamic3.hasNext()) {
-         String string = (String)dynamic3.next();
+      for (String string : GENERATOR_OPTION_KEYS) {
          Optional<? extends Dynamic<?>> optional = dynamic.get(string).result();
          if (optional.isPresent()) {
-            dynamic2 = dynamic2.set(string, (Dynamic)optional.get());
+            dynamic2 = dynamic2.set(string, (Dynamic) optional.get());
          }
       }
 
@@ -283,3 +281,5 @@
       }
    }
 }
+
+// Fukkit Marker
